### ISSUE TITLE
Building from source (zig master) results in error: enum 'std.zig.Ast.Tag' has no field named 'assign_bit_shift_left'

### ISSUE THREAD

**Issue by francisrstokes** [zigtools/zls#400]
- Linux (endeavourOS / Arch)
- x86_64
- zig compiler: stage 1, built from source (0.9.0-dev.1206+99961f22d)

```
./src/semantic_tokens.zig:823:9: error: enum 'std.zig.Ast.Tag' has no field named 'assign_bit_shift_left'
        .assign_bit_shift_left,
        ^
/home/francis/repos/zig/build/lib/zig/std/zig/Ast.zig:2481:21: note: 'std.zig.Ast.Tag' declared here
    pub const Tag = enum {
                    ^
./src/semantic_tokens.zig:284:5: note: referenced here
    switch (tag) {
    ^
./src/analysis.zig:3265:9: error: enum 'std.zig.Ast.Tag' has no field named 'assign_bit_shift_left'
        .assign_bit_shift_left,
        ^
/home/francis/repos/zig/build/lib/zig/std/zig/Ast.zig:2481:21: note: 'std.zig.Ast.Tag' declared here
    pub const Tag = enum {
                    ^
./src/analysis.zig:2698:5: note: referenced here
    switch (node_tag) {
    ^
./src/ast.zig:394:9: error: enum 'std.zig.Ast.Tag' has no field named 'assign_bit_shift_left'
        .assign_bit_shift_left,
        ^
/home/francis/repos/zig/build/lib/zig/std/zig/Ast.zig:2481:21: note: 'std.zig.Ast.Tag' declared here
    pub const Tag = enum {
                    ^
./src/ast.zig:267:18: note: referenced here
    while (true) switch (tags[n]) {
                 ^
./src/references.zig:477:9: error: enum 'std.zig.Ast.Tag' has no field named 'assign_bit_shift_left'
        .assign_bit_shift_left,
        ^
./src/references.zig:536:41: note: called from here
            try symbolReferencesInternal(arena, store, .{ .node = 0, .handle = curr_handle }, decl_handle, encoding, context, handler);
                                        ^
./src/rename.zig:39:36: note: called from here
    try references.symbolReferences(arena, store, decl_handle, encoding, true, RefHandlerContext{
                                   ^
./src/rename.zig:37:9: note: called from here
) !void {
        ^
/home/francis/repos/zig/build/lib/zig/std/zig/Ast.zig:2481:21: note: 'std.zig.Ast.Tag' declared here
    pub const Tag = enum {
                    ^
./src/references.zig:91:5: note: referenced here
    switch (node_tags[node]) {
    ^
./src/references.zig:477:9: error: enum 'std.zig.Ast.Tag' has no field named 'assign_bit_shift_left'
        .assign_bit_shift_left,
        ^
./src/references.zig:536:41: note: called from here
            try symbolReferencesInternal(arena, store, .{ .node = 0, .handle = curr_handle }, decl_handle, encoding, context, handler);
                                        ^
./src/main.zig:910:36: note: called from here
    try references.symbolReferences(
                                   ^
./src/main.zig:907:9: note: called from here
) !void {
        ^
/home/francis/repos/zig/build/lib/zig/std/zig/Ast.zig:2481:21: note: 'std.zig.Ast.Tag' declared here
    pub const Tag = enum {
                    ^
./src/analysis.zig:1684:9: error: enum 'std.zig.Ast.Tag' has no field named 'assign_bit_shift_left'
        .assign_bit_shift_left,
        ^
/home/francis/repos/zig/build/lib/zig/std/zig/Ast.zig:2481:21: note: 'std.zig.Ast.Tag' declared here
    pub const Tag = enum {
                    ^
./src/analysis.zig:1661:5: note: referenced here
    switch (tree.nodes.items(.tag)[child]) {
    ^
zls...The following command exited with error code 1:
/home/francis/repos/zig/build/bin/zig build-exe /home/francis/repos/zls/src/main.zig -OReleaseSafe --cache-dir /home/francis/repos/zls/zig-cache --global-cache-dir /home/francis/.cache/zig --name zls --pkg-begin build_options /home/francis/repos/zls/zig-cache/options/YL7Dg1RbMg2BiKJC0BTktF0xxxXT6QuYc5RSn-XjOWDi0_UY0EUjXmcU86ZSgwvX --pkg-end --pkg-begin known-folders /home/francis/repos/zls/src/known-folders/known-folders.zig --pkg-end --enable-cache 
error: the following build command failed with exit code 1:
/home/francis/repos/zls/zig-cache/o/0a9aba473c682fb13eb59bd79feb9de7/build /home/francis/repos/zig/build/bin/zig /home/francis/repos/zls /home/francis/repos/zls/zig-cache /home/francis/.cache/zig -Drelease-safe
```

**Comment by nektro** [comment]
fixed with #399

**Comment by francisrstokes** [comment]
@nektro this patch did indeed fix the issue - thanks 👍
